pdns/dnsdistdist/docs/reference/dq.rst

```diff
@@ -59,6 +71,12 @@ This state can be modified from the various hooks.
 
     :ref:`ComboAddress` of the remote client.
 
+  .. attribute:: DNSQuestion.requestorID
+
+    .. versionadded:: 1.8.0
+
+    The identifier of the requestor, which will be exported via ProtoBuf if set.
```

Comment:
in the `from` field.

Reply:
Isn't that in the `requestorId` field?

Reply:
Hmm, right. I mixed that field up with `d_requestor` used in the `m.setRequest` call. These names are too similar :-(

Reply:
You are right, `requestorId` did not exist initially but now it's confusing indeed.